xml_analyzer/xmlreader.py

```diff
@@ -1,38 +1,37 @@
 import collections
 import xml.etree.ElementTree as Et
 from .data_container import DataContainer
+from .definitions import AutocalcMethods
 
 
 class XMLReader:
-    autocalc_methods = ['autosum']
 
     def __init__(self, project_file):
         self._project_file = project_file
         self.tree = Et.parse(self._project_file)
```

Comment:
why is the project file private, but not the tree

Reply:
Fixed with https://github.com/Squiddrone/VPParametricsSolver/pull/2/commits/125a9de3f5317b3cc4f907790a9fabccc9134c0e